tokenserver-db-common/src/lib.rs

```diff
@@ -87,34 +101,41 @@ pub trait Db {
         params: params::SetUserCreatedAt,
     ) -> Result<results::SetUserCreatedAt, DbError>;
 
+    /// Update users replaced_at attribute based on user uid.
     #[cfg(debug_assertions)]
     async fn set_user_replaced_at(
         &mut self,
         params: params::SetUserReplacedAt,
     ) -> Result<results::SetUserReplacedAt, DbError>;
 
+    /// Get full user object based on passed user ID.
     #[cfg(debug_assertions)]
     async fn get_user(&mut self, params: params::GetUser) -> Result<results::GetUser, DbError>;
 
+    /// Create a complete node and return insert id from node.
     #[cfg(debug_assertions)]
     async fn post_node(&mut self, params: params::PostNode) -> Result<results::PostNode, DbError>;
 
+    /// Get complete node entry based on passed id.
     #[cfg(debug_assertions)]
     async fn get_node(&mut self, params: params::GetNode) -> Result<results::GetNode, DbError>;
 
+    /// Based on Node ID, unassign node from `users`.
     #[cfg(debug_assertions)]
-    async fn unassign_node(
+    async fn unassign_node_from_users(
```

Comment:
This is slightly better naming but the original name correlates with all the `unassign_node` references in [the python code/script](https://github.com/mozilla-services/syncstorage-rs/tree/82e84743027f3a2a703c540d055f147f00c2babe/tools/tokenserver) -- so I'd argue it's not worth changing here (as the python should also be changed) -- especially with the added docs here making it clearer (thanks for the docs btw)